docs/database.rst

```diff
@@ -206,4 +208,4 @@ Optionally you can remove left-over files which were used by the old database:
 
 .. note::
    Roundcube does not offer a migration tool for moving from SQLite to PostgreSQL.
-   In case roundcube is used, then in the setup utility SQLite can be chosen as database back end for roundcube.
+   Incase roundcube is used, the Mailu setup utility can be used to specify SQLite for the roundcube database backend.
```

Comment:
As far as I know incase is not an English word. It should be `In case roundcube ...`.

Reply:
Fixed in 58170b4f0a7926bc1ef65868a241f7224364d012